assignments/4. operators/logical-operators.md

```diff
@@ -6,55 +6,64 @@
 
 ```js
 true  && true; //output
-true  && false;
-false && true;
-false && false;
-"foo" && "bar";
-"bar" && "foo";
-"foo" && "";
-""    && "foo";
-" "   && "John" && "" && false
-false && "Hey" && undefined
-"undefined" && false && 42
+true  && false;//false
+false && true; //false
+false && false;//false
+"foo" && "bar";//"bar"
+"bar" && "foo";//"foo"
+"foo" && "";   //""
+""    && "foo";//""
+" "   && "John" && "" && false //""
+false && "Hey" && undefined   //false
+"undefined" && false && 42    //false
 ```
 
 * [ ] Logical OR operation
 ```js
-true  || true;
-true  || false;
-false || true;
-false || false;
-"foo" || "bar";
-"bar" || "foo";
-"foo" || "";
-""    || "foo";
-" "   || "John" || "" || false
-false || "Hey" || undefined
-"undefined" || false || 42
+true  || true;                  //true
+true  || false;                 //true
+false || true;                  //true
+false || false;                 //false
+"foo" || "bar";                 //"foo"
+"bar" || "foo";                 //"bar"
+"foo" || "";                    //"foo"
+""    || "foo";                 //"foo"
+" "   || "John" || "" || false  //""
+false || "Hey" || undefined     //"hey"
+"undefined" || false || 42      //"undefined"
 ```
 
 2. 🥈You have two variables i.e `isGuestOneVeg` and  `isGuestTwoVeg` according to the value using logical && and || opeartor do the following.
 
 * [ ] If both are veg "Only offer up vegan dishes."
 * [ ] At least one veg  "Make sure to offer up some vegan options."
 * [ ] Else, "Offer up anything on the menu"
-```js
+
+
 let isGuestOneVeg = false;
 let isGuestTwoVeg = false;
-// Your code goes here
-```
+
+(isGuestOneVeg == true && isGuestTwoVeg == true) && alert("Only offer up vegan dishes.");
```

Comment:
The value will always be either `true` or `false` so you don't need to check the values are equal or not.